.github/workflows/ci.yml

```diff
@@ -12,9 +12,16 @@ jobs:
       fail-fast: false
       matrix:
         runs-on: [ubuntu-latest, windows-latest]
-    name: ${{ matrix.runs-on }}
+        sdk: ["6.0.301", "6.0.302", "6.0.303", "6.0.400"]
```

Comment:
Out of curiosity, what is the significance of running CI builds for several patch versions of .NET 6? Why not just pick the most recent one of those listed here?

Reply:
The reason is to be sure that public api set is the same for all of them.